content/guides/development.md

```diff
@@ -5,182 +5,247 @@ contributors:
 
-When it's done installing, you should be able to use `webpack-dev-server` like this:
+__src/print.js__
 
-```bash
-webpack-dev-server --open
+``` diff
+  export default function printMe() {
+-   cosnole.log('I get called from print.js!');
++   console.log('I get called from print.js!');
+  }
 ```
 
-T> If your console says it can't find the command, try running `node_modules/.bin/webpack-dev-server`. Optimally you would add the command to your `package.json`, like this: `"scripts": { "start": "webpack-dev-server" }`.
-
-The command above should automatically open your browser on `http://localhost:8080`.
-
-Make a change in one of your files and hit save. You should see that the console is recompiling. After that's done, the page should be refreshed. If nothing happens in the console, you may need to fiddle with [`watchOptions`](/configuration/dev-server#devserver-watchoptions-).
-
-Now you have live reloading working, you can take it even a step further: Hot Module Replacement. This is an interface that makes it possible to swap modules **without a page refresh**. See the [Hot Module Replacement guide](/guides/hot-module-replacement) for more information.
+Now save your file and watch your command line, you should see webpack automatically recompile your code (but only the affected files)!
 
-By default **inline mode** is used. This mode injects the client - needed for live reloading and showing build errors - in your bundle. With inline mode you will get build errors and warnings in your DevTools console.
+The only downside is that you have te refresh your browser in order to see the changes. It would be much nicer if that would happen automatically as well, so let's take care of that.
 
-webpack-dev-server can do many more things such as proxying requests to your backend server. For more configuration options, see the [**devServer documentation**](/configuration/dev-server).
 
+### Using webpack-dev-server
 
-### webpack-dev-middleware
+The `webpack-dev-server` provides you with a simple web server and the ability to use live reloading. Let's set it up:
 
-webpack-dev-middleware works for connect-based middleware stacks. This can be useful if you already have a Node.js server or if you want to have full control over the server.
-
-The middleware will cause webpack to compile files in-memory. When a compilation is running, it will delay the request to a file until the compilation is done.
-
-W> This is intended for advanced users. webpack-dev-server is much easier to use.
+``` bash
+npm install --save-dev webpack-dev-server
+```
 
-Start with installing the dependencies from npm:
+Change your config file to tell the dev server where to look for files:
+
+__webpack.config.js__
+
+``` diff
+  const path = require('path');
+  const HtmlWebpackPlugin = require('html-webpack-plugin');
+
+  module.exports = {
+    entry: {
+      app: './src/index.js',
+      print: './src/print.js'
+    },
+    devtool: 'inline-source-map',
```

Comment:
Make sure you keep this in sync with whatever you decide above. Also it seems like this config changed a lot more (at least with the `HtmlWebpackPlugin`), maybe update the one above and be clear about which guide is the prequisite for this one.